packages/signers/package.json

```diff
@@ -65,6 +68,8 @@
     "viem": "^1.16.2"
   },
   "optionalDependencies": {
+    "@turnkey/http": "^2.4.2",
+    "@turnkey/viem": "^0.4.2",
```

Comment:
is `@turnkey/api-key-stamper` not required here too?

Reply:
the stamper packages are only used in the examples and tests, which is why i ultimately opted to place it under `devDependencies` but not in `optionalDependencies`. it's up to the application whether they want to use the webauthn, api key, or email auth stamper. even though these stamper dependencies aren't required for this package, i'll defer to you all as to whether it makes the developer experience smoother to add it here.